CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Allow identifying assets in code from their path (#1177. **@GalaxyCrush**).
 - Added an Audio asset (#230, **@Dageus**, **@diogomsmiranda**).
 - Hold function for debug_camera (#1030, **@jdbaracho**).
+- Added UIText component (#1300, **@mkuritsu**)
```

Comment:
```suggestion
- UI text element using MSDF text rendering (#1300, **@mkuritsu**)
```
This makes it seem like a small feature!

Reply:
Also, you can add an entry to the fixes section with something like `- Stopped undeterministic behavior when drawing transparent UI elements by drawing layer by layer (**@mkuritsu**)`